### PR TITLE
drivers:temperature:max31865pmb1: add enchancements

### PR DESCRIPTION
**Pull request by jsanbuen**
Add provisions for N-wire, filter, and RC time constant indicators.

### CODE REVIEW

**Review by amiclaus — COMMENTED** [review]
even though you splitted into 2 PRs, this commit seems to contain also the formatting changes.

**Comment by jsanbuen** [comment]
> even though you splitted into 2 PRs, this commit seems to contain also the formatting changes.

Do I need to apply the enhancements on the version that doesn't have the formatting changes or the version that is currently in the master branch?

**Comment by jsanbuen** [comment]
Reverted to version currently in master and added enhancements code, without formatting corrections, on said version for this pull request.

File: drivers/temperature/max31865pmb1/max31865.h [hunk 1]
```diff
@@ -81,6 +81,9 @@
  */
 struct max31865_dev {
 	struct no_os_spi_desc *comm_desc;
+	bool is_filt_50;
+	bool is_odd_wire;
```
[the remark this answers is not in the document]
Comment (jsanbuen) [reply drivers/temperature/max31865pmb1/max31865.h:85]: added is_filt_50 and is_odd_wire usage in functions for filter enable and N-wire configuration respectively.

**Review by amiclaus — APPROVED** [review]

**Pull request merged**